docs/application_management.md

```diff
@@ -1,27 +1,94 @@
 # Application Management
 
-Currently application creation/managment happens via the ArgoCD UI. In the future
-we will create and manage applications declaratively.
-
-## Deploying Applications with ArgoCD
-
-1. Log into ArgoCD ui at ```echo https://$(oc get routes | grep argocd-server | awk 'NR==1{print $2}')```
-2. Click create application
-3. Fill the fields out like so:
-
-```text
-Application name: <application_name>
-Project: <your_team_project>
-Sync Policy: Automatic (or manual)
-Sync Option: Check Use A Schema...
-Repository URL: <git_url>
-Revision: <branch_name> or HEAD
-Path: <path_to_overlay> or .
-Cluster: Your cluster address
-Namespace: <application_namespace>
+While ArgoCD allows you to create ArgoCD applications via the UI and CLI, it is our
```

Comment:
This doesn’t have to be done as part of this change, but is it possible to enforce this policy in ArgoCD so that users _cannot_ make changes via the UI? (if so I’d propose to add a task to our backlog to make such a change)

Reply:
Yep we can prevent people from creating new projects via ui/cli (will add to backlog)